.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
This file can be deleted, as `package-lock.json` is ignored by `.gitignore`

Reply:
But this tells npm that we don't want it to work with lock file, so npm doesn't try to write anything to it.